### PR TITLE
Fixes to make apps work on 2.8

### PR DESCRIPTION
**Pull request by rp-julienguery**
Ran all the tests with the following changes.

### CODE REVIEW

File: Netlens/src/test/java/co/cask/cdap/apps/netlens/app/NetlensAppTest.java [hunk 1]
```diff
@@ -82,7 +83,6 @@ public void testNetlensApp() throws Exception {
       }
     } finally {
       TimeUnit.SECONDS.sleep(1);
-      RuntimeStats.clearStats("");
```
Comment (bdmogal) [inline Netlens/src/test/java/co/cask/cdap/apps/netlens/app/NetlensAppTest.java:85]: don't need this method anymore?

**Comment by bdmogal** [comment]
quick question, LGTM otherwise

File: Netlens/src/test/java/co/cask/cdap/apps/netlens/app/NetlensAppTest.java (hunk 1 above)
Comment (rp-julienguery) [inline Netlens/src/test/java/co/cask/cdap/apps/netlens/app/NetlensAppTest.java:85]: The method has been deprecated, with no new usage provided. And the tests still work without it (I ran mvn clean package).

**Comment by bdmogal** [comment]
:+1:

**Pull request merged**